executor/executor_runner.h

```diff
@@ -575,6 +611,7 @@ class Runner
 	std::vector<std::string> leak_frames_;
 	int restarting_ = 0;
 	bool corpus_triaged_ = false;
+	uint64 proc_counter_ = 0;
```

Comment:
Nit: sounds more like `exec_count_` or something like that.

Reply:
+1